client/my-sites/stats/stats-upsell-modal/index.tsx

```diff
@@ -133,44 +134,94 @@ export default function StatsUpsellModal( { siteId }: { siteId: number } ) {
+					) : (
+						<PersonalFeatures plan={ plan } />
+					) }
 				</div>
 			</div>
 		</Modal>
 	);
 }
+
+function PremiumFeatures( { plan }: { plan: PlanNext | undefined } ) {
+	const translate = useTranslate();
+
+	return (
+		<div className="stats-upsell-modal__features">
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( 'All stats available: traffic trends, sources, optimal time to post…' ) }
+				</div>
+			</div>
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( 'Track campaign performance with UTM parameters' ) }
+				</div>
+			</div>
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( 'Instant access to upcoming features' ) }
+				</div>
+			</div>
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( '14-day money-back guarantee' ) }
+				</div>
+			</div>
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( 'All %(planName)s plan features', {
+						args: { planName: plan?.productNameShort ?? '' },
+					} ) }
+				</div>
+			</div>
+		</div>
+	);
+}
+
+function PersonalFeatures( { plan }: { plan: PlanNext | undefined } ) {
+	const translate = useTranslate();
+	return (
+		<div className="stats-upsell-modal__features">
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( 'Stats beyond the last 7 days' ) }
+				</div>
+			</div>
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( 'Download data as CSV' ) }
+				</div>
+			</div>
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( 'Detailed traffic stats and site insights' ) }
+				</div>
+			</div>
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( '14-day money-back guarantee' ) }
+				</div>
+			</div>
+			<div className="stats-upsell-modal__feature">
+				<Gridicon icon="checkmark" size={ 18 } />
+				<div className="stats-upsell-modal__feature-text">
+					{ translate( 'All %(planName)s plan features', {
+						args: { planName: plan?.productNameShort ?? '' },
+					} ) }
+				</div>
```

Comment:
I made some educated guesses about the copy here but after posting the PR and testing I think we should align this copy with the in-page modal.